ros_speech_recognition/requirements.txt

```diff
@@ -1 +1,2 @@
-SpeechRecognition==3.8.1
+SpeechRecognition==3.9.0
+vosk==0.3.45
```

Comment:
I am very sorry for too late comment.
Both `SpeechRecognition==3.9.0` and `vosk` are not compatible with python 3.4, so the indigo test will surely fail.

Reply:
1. Drop indigo support like https://github.com/jsk-ros-pkg/jsk_3rdparty/commit/2c9e010de62de663aa4878bba93c2f472ebf8d71 
2. Support indigo like https://github.com/jsk-ros-pkg/jsk_3rdparty/commit/e8742b036cd976e807f109018e0697dd25b14ccd

I think it may be ok to drop indigo support dependent on https://github.com/jsk-ros-pkg/jsk_3rdparty/issues/471 discussion.